templates/cli/lib/commands/init.js.twig

```diff
@@ -156,21 +159,24 @@ const initFunction = async () => {
     })
 
     fs.mkdirSync(functionDir, "777");
+    fs.mkdirSync(templatesDir, "777");
 
-    let gitInitCommands = "git clone -b v3 --single-branch --depth 1 --sparse https://github.com/{{ sdk.gitUserName }}/functions-starter ."; // depth prevents fetching older commits reducing the amount fetched
+    let gitInitCommands = "git clone --single-branch --depth 1 --sparse https://github.com/{{ sdk.gitUserName }}/templates ."; // depth prevents fetching older commits reducing the amount fetched
+
+    let gitPullCommands = `git sparse-checkout add ${answers.runtime.name}`;
```

Comment:
I think this doesnt work, because showidn templates is slow so it maybe clones everything. Ill check

Reply:
Runtime `name` is a new field, and it's just node.

Reply:
Hmm okie, Ill try to run with timings to see what takes 2-3 seconds